README.md

```diff
@@ -154,6 +165,17 @@ message PromoteVolumeResponse{
 }
 ```
 
+#### Error Scheme
+
+| Condition | gRPC Code | Description | Recovery Behavior |
+|-------------------|-------------------|-------------------|-------------------|
+| Missing required field | INVALID_ARGUMENT | Indicates that a required field is missing from the request. | Caller MUST fix the request by adding the missing required field before retrying. |
+| Volume does not exist | NOT_FOUND | Indicates that a volume corresponding to the specified `volume_id` does not exist. | Caller MUST verify that the `volume_id` is correct and that the volume is accessible and has not been deleted before retrying with exponential back off. |
+| Volume in use | FAILED_PRECONDITION | Indicates that the volume corresponding to the specified `volume_id` could not be promoted due to failed precondition (for example mirroring is not enabled). | Caller SHOULD ensure that mirroring is enabled. |
```

Comment:
could this be `volume not available` condition rather than `volume in use`?

Reply:
It will be ambiguous IMO, people can think that volume is not available in the cluster

Reply:
volume not available/not found is documented in the second point? and also it cannot be volume in use rather it should be Volume is not replicated?

Reply:
done